std-bits/base/src/main/java/org/enso/base/Text_Utils.java

```diff
@@ -35,6 +35,64 @@ public static String substring(String string, int from, int to) {
     return string.substring(from, to);
   }
 
+  /**
+   * Gets the first Grapheme cluster in the string.
+   *
+   * @param string the string to substring
+   * @return the first grapheme cluster in the string or null if the string is empty.
+   */
+  public static String first_cluster(String string) {
+    BreakIterator breakIterator = BreakIterator.getCharacterInstance();
+    breakIterator.setText(string);
+    int start = breakIterator.first();
+    if (start == -1) {
+      return null;
+    }
+    int end = breakIterator.next();
+    return string.substring(start, end);
+  }
+
+  /**
+   * Gets the last Grapheme cluster in the string.
+   *
+   * @param string the string to substring
+   * @return the last grapheme cluster in the string or null if the string is empty.
+   */
+  public static String last_cluster(String string) {
+    BreakIterator breakIterator = BreakIterator.getCharacterInstance();
+    breakIterator.setText(string);
+    int start = breakIterator.last();
+    if (start == -1) {
+      return null;
+    }
+    int end = breakIterator.previous();
+    return string.substring(end, start);
+  }
```

Comment:
Unless I'm mistaken, these two are equivalent to `take_prefix(str, 1)` and `take_suffix(str, 1)` and they should also do the same "amount of work" so it feels a bit redundant to add them.

Happy to keep them if there's some reason for keeping separate.